parsons/ngpvan/people.py

```diff
@@ -10,69 +10,80 @@ def __init__(self, van_connection):
 
         self.connection = van_connection
 
-    def find_person(self, first_name=None, last_name=None, date_of_birth=None, email=None,
-                    phone=None, street_number=None, street_name=None, zip=None, match_map=None):
+    def find_person(self, id=None, id_type=None, expand_fields=None, firstName=None, lastName=None,
+                    dateOfBirth=None, email=None, phone=None, address1=None, zip=None,
```

Comment:
This is change that should be made throughout, but we should change the variable naming convention to adhere with the rest of parsons. `firstName = first_name, dateOfBirth = date_of_birth`

I'm not sure that you need it, but there is a utility function that I wrote to convert to json variable formatting when they are strings. [parsons.utilities.json_format](https://github.com/move-coop/parsons_public/blob/master/parsons/utilities/json_format.py)

Not going to markup the rest of the PR, but should be changed throughout.